wger/settings_global.py

```diff
@@ -128,11 +132,18 @@
     # Django mobile
     'django_mobile.middleware.MobileDetectionMiddleware',
     'django_mobile.middleware.SetFlavourMiddleware',
+
+    #Social Auth
+    'social_django.middleware.SocialAuthExceptionMiddleware',
 )
 
 AUTHENTICATION_BACKENDS = (
+    'wger.utils.helpers.EmailAuthBackend',
+    'social_core.backends.open_id.OpenIdAuth',
+    'social_core.backends.google.GoogleOAuth2',
+    'social_core.backends.twitter.TwitterOAuth',
+    'social_core.backends.facebook.FacebookOAuth2',
     'django.contrib.auth.backends.ModelBackend',
```

Comment:
'django.contrib.auth.backends.ModelBackend' should be the first thing, the order is important, the auth model should run first

Reply:
Order adjusted